new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Application.java

```diff
@@ -119,4 +123,48 @@ public static boolean appAccessibleInPod(
       return false;
     }
   } 
+  
+  /**
+   * Check if the given WebLogic credentials are valid by using the credentials to 
+   * invoke a RESTful Management Services command.
+   *
+   * @param host hostname of the admin server pod
+   * @param podName name of the admin server pod
+   * @param namespace name of the namespace that the pod is running in
+   * @param username WebLogic admin username
+   * @param password WebLogic admin password
+   * @return true if the RESTful Management Services command succeeded
+   **/
+  public static boolean credentialsValid(
+      String host,
+      String podName,
+      String namespace,
+      String username,
+      String password) {
+    int adminServiceNodePort = getServiceNodePort(
+        namespace, podName + "-external", WLS_DEFAULT_CHANNEL_NAME);
+
+    if (username == null) {
+      username = ADMIN_USERNAME_DEFAULT;
+    }
+    if (password == null) {
+      password = ADMIN_PASSWORD_DEFAULT;
+    }
+    StringBuffer cmdString = new StringBuffer()
+        .append("status=$(curl --user " + username + ":" + password)
+        .append(" http://" + host + ":" + adminServiceNodePort)
+        .append("/management/tenant-monitoring/servers/managed-server1")
+        .append(" --silent --show-error ")
+        .append(" -o /dev/null")
+        .append(" -w %{http_code});")
+        .append("echo ${status}");
```

Comment:
instead of using curl, you can use the adminNodePortAccessible test assertion to  validate the new credentials.

Reply:
I am trying to do a REST call that causes the admin server to forward the request to a managed server. I believe adminNodePortAccessible only connects to the admin server.

Reply:
oh, I guess that's good enough.